web/packages/studio/env/.env.dev.local.sample

```
@@ -20,6 +20,7 @@ VITE_AUTH_SCOPE_PREFIX="api://<your-client-id>"
 
 # Feature Flags (VITE_FF_* prefix)
 # See src/constants/featureFlags.ts for available flags and documentation.
+VITE_FF_AGENT_OVERVIEW_ENABLED='false'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)